libbeat/api/server.go

```diff
@@ -24,6 +24,7 @@ func Start(cfg *common.Config) {
 
 		// register handlers
 		mux.HandleFunc("/", rootHandler())
+		mux.HandleFunc("/state", stateHandler)
```

Comment:
I wonder if it would make more sense to call this `/modules`, do you plan to add anything else?

Reply:
I'm thinking of also adding information about the datasets, the output used. But not sure where it will evolve into. We probably need soon a more general discussion about api endpoint naming and structure.

Reply:
yes, API endpoints are not documented so far and we can still change them.